app/views/contracts/_contracts_table.html.erb

```diff
@@ -0,0 +1,49 @@
+<div class="content">
+        <div class="material-datatables">
+          <table id="contract_table" class="table table-no-bordered table-hover" >
+            <thead>
+              <tr>
+                <th>Código</th>
+                <th>Código de Cotización</th>
+                <th>Laboratorio</th>
+                <th>Fecha de Registro</th>
+                <th>Estado</th>
+                <th>Acciones</th>
+              </tr>
+            </thead>
+            <tbody>
+              <% contract.each do |c| %>
+                <tr>
+                  <td><%= c.code %></td>
+                  <td><%= c.code_Quotations %></td>
+                  <td><%= c.laboratory %></td>
+                  <td><%= c.date %></td>
+                  <td><%= c.state %></td>
+                  <td>
+                    <%= link_to '<i class="fa fa-eye" aria-hidden="true"></i>'.html_safe, contract_path(contract),class: "btn btn-primary btn-sm",title: "Ver", method: :delete %>
+                  </td>
```

Comment:
Use the <%= link_to do %> notation to wrap html within an anchor ( a ) tag.

See this reference: https://stackoverflow.com/questions/1086360/how-do-i-wrap-link-to-around-some-html-ruby-code